store/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -305,7 +309,7 @@ public void testWriteAndReadFilesNonTransactional() throws IOException, Interrup
     // Write to a Non Transactional Table
     TestUtil.writeFilesAndVerify(new Schema(fields), path, true, false);
 
-    CarbonReader reader = CarbonReader.builder(path, "_temp")
+    CarbonReader reader = CarbonReader.builder(path, "_temp").isTransactionalTable(true)
```

Comment:
There has invoked isTransactionalTable method in line 314, why invoke again?

Reply:
This came by conflict resolve. This will be removed in the next commit. No harm.

Reply:
This is a conflict resolve problem. Will be removed in the next commit.  Finally it should be false for this testcase

Reply:
ok, It's fine.